pycaption/microdvd.py

```diff
@@ -0,0 +1,105 @@
+from copy import deepcopy
+
+from .base import (
+    BaseReader, BaseWriter, CaptionSet, CaptionList, Caption, CaptionNode, DEFAULT_LANGUAGE_CODE)
+from .exceptions import CaptionReadNoCaptions, InvalidInputError
+import re
+
+
+class MicroDVDReader(BaseReader):
+    def detect(self, content):
+        return re.match("{\d+}{\d+}", content) is not None
+
+    def read(self, content, lang=DEFAULT_LANGUAGE_CODE):
+        if type(content) != str:
+            raise InvalidInputError('The content is not a unicode string.')
+
+        lines = content.splitlines()
+        captions = CaptionList()
+        fps = 25.0
+        for line in lines:
+            m = re.match(r"{(\d+)}{(\d+)}(.*)", line)
+            if not m: break
+
+            start, end, txt = m.groups()
+
+            if start == '0' and end == '0':
+                fps = float(txt)
```

Comment:
This part looks like not handled in the example. It could throw a TypeError if we have invalid:
"{0}{0}text not float"

Reply:
@dVeza Please have a look.